PhoneNumberKit/PartialFormatter.swift

```diff
@@ -93,13 +93,23 @@ public final class PartialFormatter {
      - returns: Formatted phone number string.
      */
     public func formatPartial(_ rawNumber: String) -> String {
+        // Verify if number contains Arabic
+        let predicate = NSPredicate(format: "SELF MATCHES %@", "(?s).*\\p{Arabic}.*")
+        let containsArabic = predicate.evaluate(with: rawNumber)
+
+        // Convert number to latin
+        var latinRawNumber = rawNumber
+        if #available(iOSApplicationExtension 9.0, *) {
+            latinRawNumber = rawNumber.applyingTransform(.toLatin, reverse: false) ?? rawNumber
+        }
+
         // Always reset variables with each new raw number
         self.resetVariables()
 
-        guard self.isValidRawNumber(rawNumber) else {
-            return rawNumber
+        guard self.isValidRawNumber(latinRawNumber) else {
+            return latinRawNumber
```

Comment:
Why would you return the latin converted number here?  Shouldn't it leave it in its original format?

Reply:
yeah, probably rawNumber is better here... will verify.